src/index.html

```diff
@@ -6,6 +6,10 @@
   <base href="/">
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <link rel="icon" type="image/x-icon" href="favicon.ico">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/KaTeX/0.7.1/katex.min.css">
```

Comment:
This index.html is not the best place to place these kinds of assets in an angular context. The css would need to be added as an angular import, which would mean adding it to our `styles.scss` file, as described [here](https://www.npmjs.com/package/ng-katex#important)